ext/openssl/openssl.c

```diff
@@ -933,6 +933,44 @@ static void php_openssl_dispose_config(struct php_x509_request * req) /* {{{ */
 }
 /* }}} */
 
+static ENGINE *php_openssl_make_pkcs11_engine(const bool warn) /* {{{ */
+{
+	char *verbose = NULL;
+	ENGINE *engine;
+
+	engine = ENGINE_by_id("pkcs11");
+	if (engine == NULL) {
+		if (warn)
```

Comment:
There should be `{` - see https://github.com/php/php-src/blob/master/CODING_STANDARDS.md#syntax-and-indentation

Reply:
Was done https://github.com/php/php-src/commit/27d8f44af90e66dcdb423979a15dc91f4d7f83e4